build.gradle

```diff
@@ -85,6 +86,31 @@ allprojects {
     classifier = 'tests'
   }
 
+  distributions {
```

Comment:
Hi, has this been tested and run either at your workplace? 😄 Thanks!

Reply:
I updated it to:
 - rebase onto 2.0.7
 - depends on `jar`, so there is something in the archives even if you don't launch `jar` task before
 - intergate the new `configDocs` dir

I added the result in the PR description.

Reply:
> I updated it to:
> 
> * rebase onto 2.0.7
> * depends on `jar`, so there is something in the archives even if you don't launch `jar` task before
> * intergate the new `configDocs` dir
> 
> I added the result in the PR description.

Thanks for the rebase. Please ensure that, for people who don't need to reply on a distribution tarball/zip, they can clone the repo/download automatic zip of the releases as per usual then then build with gradle.